core/src/main/scala/org/locationtech/rasterframes/functions/package.scala

```diff
@@ -149,8 +149,8 @@ package object functions {
 
     sqlContext.udf.register("rf_cell_types", cellTypes)
     sqlContext.udf.register("rf_rasterize", rasterize)
+    // TODO: replace this with full ReprojectGeometry with String handling.
```

Comment:
as above let's get all the function registration cleaned up before merge.

Reply:
I feel like that's a separate PR.... trying to move away from my habit of monolithic PRs that take forever to review!